ibis/backends/dask/execution/aggregations.py

```diff
@@ -107,37 +117,32 @@ def execute_aggregation_dataframe(
     return result
 
 
-# TODO - aggregations - #2553
-# @execute_node.register((ops.Any, ops.All), (dd.Series, SeriesGroupBy))
-# def execute_any_all_series(op, data, aggcontext=None, **kwargs):
-#     if isinstance(aggcontext, (agg_ctx.Summarize, agg_ctx.Transform)):
-#         result = aggcontext.agg(data, type(op).__name__.lower())
-#     else:
-#         result = aggcontext.agg(
-#             data, lambda data: getattr(data, type(op).__name__.lower())()
-#         )
-#     return result
+@execute_node.register((ops.Any, ops.All), (dd.Series, ddgb.SeriesGroupBy))
+def execute_any_all_series(op, data, aggcontext=None, **kwargs):
+    if isinstance(aggcontext, (agg_ctx.Summarize, agg_ctx.Transform)):
```

Comment:
What are the two branches trying to handle differently?

Reply:
https://github.com/ibis-project/ibis/blob/master/ibis/backends/pandas/aggcontext.py as the different types of aggregation contexts that the pandas backend supports, which the dask backend currently uses. 

Right now the dask backend has no windowing, so the second branch isn't hit, but I left it in there to make starting on the windowing piece easier. I suspect we're going to have to treat this the same way.

Reply:
I see. So the aggcontext here is a pandas aggcontext? (Would be great if you can clarify this in the code)

I wonder how did it work if we are passing a dask.Series/dask.SeriesGroupBy to the pandas aggcontext...

Reply:
If the branch isn't hit/tested, perhaps put a comment I don't need to spend time reviewing it :)

Reply:
Yes - the dask backend does not currently implement its own aggegation context, but that may change when we get to more complicated operations.